WordPress/Classes/ViewRelated/NUX/SigninWPComSyncHandler.swift

```diff
@@ -0,0 +1,88 @@
+import UIKit
+import WordPressComAnalytics
+
+/// A protocol and extension encapsulating syncing behavior common to WPCom 
+/// signin controllers.  Responsible for syncing account information and blog 
+/// details of the user.
+///
+protocol SigninWPComSyncHandler: class
+{
+    func configureLoading(loading: Bool)
+    func configureStatusMessage(message: String)
```

Comment:
"configure" sounds odd when they just look like setters. I'd rather do `setStatusMessage(message: String)`. I'd prefer a property but not sure if that plays nice with ObjC `var statusMessage: String { get set }`